src/tests/test_default_fake.py

```diff
@@ -229,7 +231,7 @@ def test_const(TestData):
         assert isinstance(f["country"], str)
         assert f["country"] == "United States of America"
 
-
+#@pytest.mark.xfail(reason="Always failing with "no such file or dir" error!!")
```

Comment:
This should exist if youre running the tests from a unix based machine and from the same directory as the src folder